### ISSUE TITLE
Exception during using of anonymous types

### ISSUE THREAD

**Issue by lextudio-support** [obfuscar/obfuscar#501]
Hi.
When i try to execute code like that:
```
var elems = LastResults.SelectMany(cl => new[]
                    {
                        new
                        {
                            refno = cl.Element1,
                            db = cl.DbElement1,
                        },
                        new
                        {
                            refno = cl.Element2,
                            db = cl.DbElement2,
                        }
                    });
```
in runtime i've got an exception:
```
System.ArgumentException: Метод ' . `2[System.String,<Another_Full_Class_Name>.DbElement]. ' не является средством доступа к свойству
   в System.Linq.Expressions.Expression.GetProperty(MethodInfo mi)
   в System.Linq.Expressions.Expression.Property(Expression expression, MethodInfo propertyAccessor)
   в <Some_Full_Class_Name>.Engine. ()
```
Not obfuscated code works well. And after i've added name to the type, the code has executed well with obfuscar.

**Original Reported Date**: 2014-07-08T01:43:26.96-07:00
**Planned For Release**: 
**Original CodePlex ID**: 18

**Comment by lextudio-support** [comment]
**Copied from CodePlex without authors**:

Thanks for the sample code. I will do some analysis and get back to you.

**Original Posted Date**: 2014-07-08T22:40:10.7-07:00

**Comment by lextudio-support** [comment]
More info is required.

**Original Closed Date**: 2015-08-04T05:39:59.81-07:00
